build_packet/debian/control

```diff
@@ -0,0 +1,13 @@
+Source: cligen
+Section: unknown
+Priority: optional
+Maintainer: roma55592@yandex.ru
+Build-Depends: debhelper-compat (= 12)
+Standards-Version: 7.0.1
```

Comment:
Standars-Version should be something like 4.7.0
https://www.debian.org/doc/debian-policy/upgrading-checklist.html

Reply:
> Standars-Version should be something like 4.7.0 https://www.debian.org/doc/debian-policy/upgrading-checklist.html

@YotaYota A very good point, thank you